api/utils/utilities.js

```diff
@@ -0,0 +1,8 @@
+module.exports = () => {
```

Comment:
Any type of handler should probably be saved here in the utils file. good stuff!